src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -64,4 +66,42 @@ public List<AdvertBasicInfoDTO> getByCategory(String category) {
         }
 
     }
+
+    @Transactional
+    public void deleteAdvertById(Long id) {
+        if (!advertRepository.existsById(id)) {
+            log.error("Advert with ID {} not found", id);
```

Comment:
Add hadling of "your" exception in GlobalExceptionHandler.java and move logging messagere there
(Maksim already mentioned it erlier - https://github.com/Volodymyr95/goto-m5f/pull/21#discussion_r1829778469)